tests/test_boundless_read.py

```diff
@@ -0,0 +1,19 @@
+"""Test of boundless reads"""
+
+from hypothesis import given
+import hypothesis.strategies as st
+import numpy
+
+import rasterio
+from rasterio.windows import Window
+
+
+@given(st.integers(min_value=0, max_value=700))
+def test_outer_boundless_pixel_fidelity(size):
+    """An outer boundless read doesn't change pixels"""
+    path_rgb_byte_tif = 'tests/data/RGB.byte.tif'
```

Comment:
Note `path_rgb_byte_tif` is a fixture available from `conftest.py`, no need to define it here.

Reply:
I thought I remembered that pytest fixtures and hypothesis givens don't play well together, but I've figured it out.